cmd/chart/upgrade_test.go

```diff
@@ -0,0 +1,97 @@
+package chart
+
+import (
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+)
+
+type test struct {
+	name     string
+	image    string
+	expected string
+}
+
+func Test_ChartUpgrade(t *testing.T) {
+	tests := []test{
+		{
+			name:     "builder_v11_rootless",
```

Comment:
The name could be written in english..

"v11_rootless" doesn't help a contributor or maintainer who comes here 1 year later (or perhaps even yourself) with figuring out what this test case does.